PowerEditor/src/ScintillaComponent/ScintillaEditView.cpp

```diff
@@ -2570,9 +2570,9 @@ void ScintillaEditView::performGlobalStyles()
 	StyleArray & stylers = nppParams.getMiscStylerArray();
 
 	const Style * pStyle = stylers.findByName(TEXT("Current line background colour"));
-	if (pStyle)
+	if (pStyle && isCurrentLineHiLiting())

```

Comment:
Use `NppParameters::getInstance().getSVP()._currentLineHilitingShow;`  instead of `isCurrentLineHiLiting`.

Reply:
Done. Also incorporated @Yaron10 's suggestion assuming that the findByName has some runtime cost which can be avoided when the current line highlighting preference is off.